lib/repl.js

```diff
@@ -1044,6 +1047,33 @@ function complete(line, callback) {
     }
 
     completionGroupsLoaded();
+  } else if (line.match(functionCallRE)) {
+    const fnParenthesesIndex = line.lastIndexOf('(');
+    const fullMethod = line.slice(0, fnParenthesesIndex);
+    const paramsPassed = line.slice(fnParenthesesIndex+1) || '';
+
+    match = simpleExpressionRE.exec( fullMethod );
+
+    const expr = match[0] + '[repl.replComplete]';
+    const evalExpr = `try { ${expr} } catch (e) {}`;
+
+    this.eval(evalExpr, this.context, 'repl', (e, completer) => {
+      var completionGroup = [];
+      if(typeof completer === 'function') {
```

Comment:
nit: be sure to run `make lint` ... there needs to be a space between the `if` and the `(` here :-)